### ISSUE TITLE
Can use different folder for upload / view ?

### ISSUE THREAD

**Issue by leo1989work** [Studio-42/elFinder#1725]
For example , the folder structure is like this

/temp_upload
/my_project/upload

temp_upload any my_project is at the same level.

So, for security reason, the upload folder inside my_project has no permission, that means PHP can not mkdir / upload file/ copy / delete etc.. only can view and download file

On the other hand, the temp_upload has every permission, and there is some script to call, to make dir, move file from temp_upload to upload, rename etc....

In short, the problem is , how to set the root to /my_project/upload/  (that means can view and download file), and  handle these action e.g.

create folder
upload
copy / cut and paste
rename
move
delete

customize the action e.g. upload target to temp_upload instead of /my_project/upload , after upload success, run the script to move file to upload folder etc...

Thanks a lot for helping.

**Comment by nao-pon** [comment]
@leo1989work It is impossible. If you allow it through the connector of elFinder, the security settings of the server is no longer meaningful. Don't you think so?

**Comment by nao-pon** [comment]
There is no reply,  so close this.
